### ISSUE TITLE
example using post?

### ISSUE THREAD

**Issue by matthiasdebernardini** [cloudwalk/axum-test-helper#11]
I am struggling with creating a test that uses post, any tips?

**Comment by AlbanMinassian** [comment]
```rs
let res = client.post("/").json(serde_json::json!({})).send().await;
```

see all options: <https://docs.rs/axum-test-helper/latest/axum_test_helper/struct.RequestBuilder.html>

**Comment by perobertson** [comment]
I was struggling a bit too and found out that `multipart` was supported but `form` data was not yet supported so I added in in here with a test as an example: https://github.com/cloudwalk/axum-test-helper/pull/17

**Comment by renancloudwalk** [comment]
Closed with #17
